src/components/Widgets/Widget.tsx

```diff
@@ -199,8 +202,7 @@ export function Widget({
       buildUrl: true,
       // insurance: true,
       integrator: integratorStringByType,
-      tokens:
-        partnerName === ThemesMap.Memecoins && tokens ? { allow: tokens } : {},
```

Comment:
Are you sure about removing this? is that part of the story? memecoins page won't work anymore

Reply:
This is now being handled differently: The tokens which we want to add (such as for TAI) go into the` tokens.ts`. 
Those meme-coin tokens will only be added for certain pages and we still have that if condition. Please take a look. Let me know, if that should be done in a useMemo or somehow else.